contents/product/session-recording.mdx

```diff
@@ -35,6 +35,28 @@ documentation: /docs/user-guides/recordings
 
 <TutorialsSlider topic="session recordings" />
 
+### How does PostHog compare?
+
+|                                       | HotJar | Logrocket |  |   Matomo  |                              PostHog                              |
+|---------------------------------------|:------:|:---------:|:--------:|:---------:|:-----------------------------------------------------------------:|
+|                                                              **Platform**                                                                             |
+| Free plan                             |    ✅   |     ✅     |          |    ✅     |                                 ✅                               |
+| Open source                           |    ❌   |     ❌     |          |    ✅     |                                 ✅                               |
+| Self-hostable                         |    ❌   |     ✅     |          |    ✅     |                                 ✅                               |
+| Cloud hosting                         |    ✅   |     ✅     |          |    ✅     |                                 ✅                               |
+| Product analytics suite               |    ❌   |     ✅     |          |    ✅     |                                 ✅                               |
+|                                                              **Session Recording**                                                                    |
+| Single-page app support               |    ✅   |     ✅     |          |     ✅     |                                 ✅                              |
+| Mobile recordings                     |    ✅   |     ✅     |          |     ✅     |                                 ✅                              |
+| Identity detection                    |    ✅   |          |          |     ✅     |                                 ✅                                |
+| Target recordings by URL              |    ✅   |          |          |     ✅     |                                 ✅                                |
+| Target by sample size                 |    ❌   |          |          |     ✅     |                                 ✅                                |
+| Filter recordings by user or cohort   |    ❌   |          |          |     ✅     |                                 ✅                                |
+| Filter recordings by event            |        |          |          |           |                                 ✅                                 |
+| Privacy masking for sensitive content |    ✅   |     ✅     |          |     ✅     |                                 ✅                               |
+| Export session data                   |    ✅   |          |          |     ✅     |                                 ❌                                 |
+| Retention policy for recordings       | 1 year |    None   |      | 24 months | 3 weeks on PostHog Cloud<br/>Unlimited on self-hosted |
+
```

Comment:
One thought for another feature would be "handle users on multiple tabs". There's probably a better way of saying it, but we merge the recordings when a user moves back and forth between tabs. I'm not sure what these competitors do in this case, but some other competitors just split those recordings, so it's hard to see what the user is actually doing.

Also, there's:
* Skipping inactivity
* Adjustable playback speed

Reply:
The tab one could be interesting. Let me think on a way to term that and look at others. 

I considered the others in an earlier version (along with a more detailed filtering section), but it feels like a really small feature for comparisons and I'd like to keep these page focused on the real selling points. It feels unlikely that anyone would choose PostHog over LogRocket because we have a fast forward button. 

Open to pushback on this if there's word from users!